mteb/tasks/Retrieval/SyntecRetrieval.py

```diff
@@ -0,0 +1,53 @@
+import datasets
+
+from ...abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class SyntecRetrieval(AbsTaskRetrieval):
+    _EVAL_SPLITS = ["test"]
+
+
+    @property
+    def description(self):
+        return {
+            "name": "SyntecRetrieval",
+            "hf_hub_name": "lyon-nlp/mteb-fr-retrieval-syntec-s2p",
+            "reference": "https://huggingface.co/datasets/lyon-nlp/mteb-fr-retrieval-syntec-s2p",
+            "description": (
+                "This dataset has been built from the Syntec Collective bargaining agreement."
+                "It maps a question to an article from the agreement"
+            ),
+            "type": "Retrieval",
+            "category": "s2p",
+            "eval_splits": self._EVAL_SPLITS,
+            "eval_langs": ["fr"],
+            "main_score": "ndcg_at_5",
```

Comment:
Interesting, why not ndcg_at_10?

Reply:
The dataset is rather small (only 90 documents, for 100 questions). We have observed, that when looking at ndcg@10 all good models seem to perform the same on the task. However, ndcg@5 allows for better discrimination between model performance.

Reply:
Makes sense!